### DISCUSSION TITLE
Asking for feedback and suggestion to improve my first project coded using R via Rstudio

### DISCUSSION THREAD

**Discussion by charlenelow** [community/community#63386] [Programming Help]
### Select Topic Area

General

### Body

Hi,
I am relatively new to Data Analytics and have been self learning it through taking up a Google Data Analytics Processional Certification course on Coursera. 

I have completed my first ever Data analytic project that I published on Github. It is a very simple project where I clean, organise, transform and analysis a set of data on Food Choice and Preferences of College Students. You can refer to this link for more details regarding it: https://github.com/charlenelow/Food-Choices-and-Preferences-of-College-Students/tree/main

I would like to get some feedback and suggestions on what I can improve on! I would greatly appreciate your help for that! Thank you in advance! 

Please refer to my README.md file within the repository to get more details regarding the project.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @charlenelow ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
